### DISCUSSION TITLE
[Any method to disable set solib-search-path]

### DISCUSSION THREAD

**Discussion by hiberabyss** [microsoft/vscode-cpptools#9708] [Q&A]
The auto executed command `-gdb-set solib-search-path "/path/to/something:"` make there a problem to get breakpoint work, is there any method to just disable this command?

**Comment by hiberabyss** [comment]
Could the `setupCommands` be executed after the initial commands?

**Reply by sean-mcmanus** [reply]
@WardenGnaw Do you know about this?

**Reply by WardenGnaw** [reply]
If you have special commands that need to be run at the end of the setup / launch process, you can use `postRemoteConnectCommands`

https://github.com/microsoft/vscode-cpptools/blob/304858c0cb7ab75defe5abc18d360c257ec1ed3b/Extension/tools/OptionsSchema.json#L545-L552
